src/chat/chatter.py

```diff
@@ -369,8 +369,58 @@ def set_color(self):
             color = pcolors.getUserColor(_id, name)
         self.setForeground(QtGui.QColor(color))
 
+    def name_used_by_other(self):
+        api_link = 'https://api.faforever.com/data/player?include=names&' \
+                   'filter=(login==' + self.user.name + ',names.name==' + self.user.name + ')'
+        response = requests.get(api_link).json()
+        if response.get('errors'):
+            return ''
+        result = ''
+        if response.get('data') is not None and len(response['data']) >= 1:
+            for ply in response['data']:
+                if ply['type'] == 'player':
+                    if ply['attributes']['login'] != self.user.name:
+                        result += (ply['attributes']['login'] + '<br/>')
+        if len(result) > 1:
+            result = 'The name has previously been used by :<br/><br/>' + str(result)
+        else:
+            result = 'The name has never been used by anyone else.'
+        return result
+
+    def names_previously_known(self):
+        api_link = 'https://api.faforever.com/data/player?include=names&fields[player]=login,names&' \
+                   'fields[nameRecord]=name,changeTime&&filter[player]=login==' + self.user.name
+        response = requests.get(api_link).json()
+        if response.get('errors'):
+            return ''
+        if response.get('included') is not None and len(response['included']) >= 1:
+            nick_list = 'The player has previously been known as :' \
+                        '<br/><table border="0" cellpadding="0" cellspacing="0" width= "180"><tbody>' \
+                        '<tr><th align="left">Name</th><th align="right">used until</th></tr>'
+            for nicks in response['included']:
+                if nicks['type'] == 'nameRecord':
+                    nick_list += '<tr><td>{}</td><td align="right">{}</td></tr>'\
+                        .format(nicks['attributes']['name'], nicks['attributes']['changeTime'][:10])
+            nick_list += '<tr><td>{}</td><td align="right">currently used</td></tr>' \
+                         '</tbody></table><br/>'.format(self.user.name)
+        else:
+            nick_list = 'The name is not currently owned by any player.'
+        return nick_list
+
     def viewAliases(self):
-        QtGui.QDesktopServices.openUrl(QUrl("{}?name={}".format(Settings.get("USER_ALIASES_URL"), self.user.name)))
+        # QtGui.QDesktopServices.openUrl(QUrl("{}?name={}".format(Settings.get("USER_ALIASES_URL"), self.user.name)))
```

Comment:
Don't leave commented-out code, we can recover it with git if need be.

Reply:
ok